examples/docker_compose_with_mongodb/app.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from pymongo import MongoClient
+import hug
+
+client = MongoClient('db', 27017)
```

Comment:
There should be 2 new lines after import statements 👍

Reply:
Thanks!